recipe/meta.yaml

```
@@ -31,7 +31,7 @@ test:
 
 about:
   home: https://github.com/ICRAR/crc32c
-  license: GNU Lesser General Public v2 or later (LGPLv2+)
+  license: LGPL-2.0-or-later
```

The previous config was not quite right, this should be
```suggestion
  license: LGPL-2.1-or-later
```
according to
https://github.com/ICRAR/crc32c/blob/v2.7.1/setup.py#L58
or
https://github.com/ICRAR/crc32c/blob/v2.7.1/LICENSE#L1-L2